INSTALL.md

```diff
@@ -36,62 +36,49 @@ to manage Accumulo:
-targeted for your environment, these files still require a few more edits before starting Accumulo.
+Follow the steps below to configure `accumulo-env.sh`:
 
-### Secret
+1. Set `HADOOP_PREFIX` and `ZOOKEEPER_HOME` to help Accumulo locate Hadoop and Zookeeper jars
+   and add them to the `CLASSPATH` variable. If you are running a vendor-specific release of Hadoop
+   or Zookeeper, see the `Vendor-specific configuration` documentation in the Administration section
+   of the Accumulo user manual as you may need to change how your `CLASSPATH` is built. If Accumulo
+   has problems later on finding jars, run `accumulo classpath -d` to print Accumulo's classpath.
 
-Accumulo coordination and worker processes can only communicate with each other
-if they share the same secret key.  To change the secret key set
-`instance.secret` in `accumulo-site.xml`.  Changing this secret key from
-the default is highly recommended.
+2. Accumulo tablet servers are configured by default to use 1GB of memory (768MB is allocated to
+   JVM and 256MB is allocated for native maps). Native maps are allocated memory equal to 33% of
+   the tserver JVM heap. The table below can be used if you would like to change tsever memory
+   usage in the `JAVA_OPTS` section of `accumulo-env.sh`:
 
-### Dependencies
+    | Native? | 512MB             | 1GB               | 2GB                 | 3GB           |
+    |---------|-------------------|-------------------|---------------------|---------------|
+    | Yes     | -Xmx384m -Xms384m | -Xmx768m -Xms768m | -Xmx1536m -Xms1536m | -Xmx2g -Xms2g |
+    | No      | -Xmx512m -Xms512m | -Xmx1g -Xms1g     | -Xmx2g -Xms2g       | -Xmx3g -Xms3g |
 
-Accumulo requires running [Zookeeper][3] and [HDFS][4] instances.  Also, the
-Accumulo binary distribution does not include jars for Zookeeper and Hadoop.
-When configuring Accumulo the following information about these dependencies
-must be provided.
-
- * **Location of Zookeepers** :  Provide this by setting `instance.zookeeper.host`
-   in `accumulo-site.xml`.
- * **Where to store data** :  Provide this by setting `instance.volumes` in
-   `accumulo-site.xml`.  If your namenode is running at 192.168.1.9:9000
-   and you want to store data in `/accumulo` in HDFS, then set
-  `instance.volumes` to `hdfs://192.168.1.9:9000/accumulo`.
- * **Location of Zookeeper and Hadoop jars** :  Setting `ZOOKEEPER_HOME` and
-   `HADOOP_PREFIX` in `accumulo-env.sh` will help Accumulo find these jars
-   when using the default setting for `general.classpaths` in accumulo-site.xml.
-
-If Accumulo has problems later on finding jars, then run `bin/accumulo
-classpath` to print out info about where Accumulo is finding jars.  If the
-settings mentioned above are correct, then inspect `general.classpaths` in
-`accumulo-site.xml`.
+3. (Optional) The Accumulo master is configured by default to use 512MB while the garbage collector
```

Comment:
Can you word this differently so that the exact memory amounts are not specified here?  If those defaults are changed in the env file, then the person making the change would need to know to make the changes here.  I think its better to not duplicate the info here.